sections/design.tex

```diff
@@ -11,7 +11,7 @@ \section*{Design considerations}
 Workspaces must also accommodate interactive project work where small teams can gather and work together uninterrupted by other groups.    
 Universities are an obvious first choice for hosting a hack week given their access to scientific resources, research support and infrastructure. 
 However traditional university lecture halls often do not meet the hack week criteria for interactive exchanges.
-Fortunately active learning teaching methods are creating new environments that foster interactivity through modified seating arrangements and integration of computer monitors at group workstations.
+Fortunately active learning teaching methods are creating new environments that foster interactivity through modified seating arrangements and integration of computer monitors at group workstations. There is a natural tension between keeping the group together and providing physically separated break-out spaces. On the one hand, conducting the workshop in a single large venue improves group cohesion and prevents self-segregation of researchers and ideas. On the other hand, the interactive nature of the workshop may lead to an environment where it is difficult for individuals to focus on the highly complex tasks that are typical for hack week projects. It may hence be advisable to allow for diffusion of the group into adjacent rooms, while providing ample venue for the entire group to congregate for tutorials, breaks and reports.
```

Comment:
I find this first sentence to be a bit cryptic. If I didn't know the specific case this refers to, this would just confuse me. I realize it was here before. How about: "Fortunately, many universities are experimenting with other types of spaces: for example, the adoption of active learning teaching methods [CITE: http://onlinelibrary.wiley.com/doi/10.1002/j.2168-9830.2004.tb00809.x/full] have led to the development of modular classrooms, where seating arrangements can be flexibly modified and group activities can be more readily undertaken".

Reply:
👍